Resources/config/mongodb.xml

```diff
@@ -32,7 +32,6 @@
         <!-- metadata -->
         <parameter key="doctrine_mongodb.odm.metadata.driver_chain.class">Doctrine\Persistence\Mapping\Driver\MappingDriverChain</parameter>
         <parameter key="doctrine_mongodb.odm.metadata.attribute.class">Doctrine\ODM\MongoDB\Mapping\Driver\AttributeDriver</parameter>
```

Comment:
This 2 parameters are no longer necessary. 
```suggestion
```

You should also remove the assertions on this parameters in the tests.

Reply:
They are still necessary, the thing about how it works is that (for `$driverType = 'attribute'` for example) [this code](https://github.com/symfony/symfony/blob/fec89090f178f32b686099bec5b2d1521673c514/src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php#L205-L207):

```php
$mappingDriverDef = new Definition($this->getMetadataDriverClass($driverType), [
    array_values($driverPaths),
]);
```
calls to [our](https://github.com/doctrine/DoctrineMongoDBBundle/blob/3c499fb71ab4df6e20cce271e86d3670631c6fd4/DependencyInjection/DoctrineMongoDBExtension.php#L547-L550):
```php
protected function getMetadataDriverClass(string $driverType): string
{
    return '%' . $this->getObjectManagerElementName('metadata.' . $driverType . '.class') . '%';
}

// ...

protected function getObjectManagerElementName(string $name): string
{
    return 'doctrine_mongodb.odm.' . $name;
}
```
so it looks for `%doctrine_mongodb.odm.metadata.attribute.class%`

Reply:
Ok, got it.